components/prism-keyman.js

```diff
@@ -0,0 +1,14 @@
+Prism.languages.keyman = {
+	'comment': /\b[cC]\s.*?(\r?\n|$)/,
+	'function': /\[\s*((CTRL|SHIFT|ALT|LCTRL|RCTRL|LALT|RALT)\s+)*(([T|K|U]_[a-z0-9_]+)|(".+")|('.+'))\s*\]/i,  // virtual key
```

Comment:
I think you meant `[TKU]` instead of `[T|K|U]`, even though I can't find any usage of `T` or `U` in the documentation...

Furthermore, if I'm reading this properly, you don't need the three pairs of inner parentheses in the last part: `([T|K|U]_[a-z0-9_]+|".+"|'.+')`.

Also, it looks like you forgot the `CAPS` and `NCAPS` shift codes.

Finally, the documentation mentions key codes like `K_KANJI?15` which do not appear to be handled here (due to the `?` char)